README.adoc

```diff
@@ -1,6 +1,6 @@
 image:https://img.shields.io/clojars/v/io.github.pfeodrippe/pitoco.svg[link="http://clojars.org/io.github.pfeodrippe/pitoco",title="Clojars Project"]
 
-= Pitoco
```

Comment:
I'm not an asciidoc user, but my asciidoc engine says:
```
level 0 sections can only be used when doctype is book
```
Adding an extra `=` fixes this issue.
If you prefer, I can rollback this change